massimo/patterns/header.php

```diff
@@ -0,0 +1,41 @@
+<?php
+/**
+ * Title: header
+ * Slug: massimo/header
+ * Categories: hidden
+ * Inserter: no
+ */
+?>
+<!-- wp:group {"align":"wide","style":{"border":{"color":null,"style":null,"width":null,"top":{"width":"2rem"},"right":[],"bottom":[],"left":[]},"spacing":{"blockGap":"var:preset|spacing|70","padding":{"top":"var:preset|spacing|40","bottom":"var:preset|spacing|70"}}},"layout":{"type":"constrained"}} -->
+<div class="wp-block-group alignwide" style="border-top-width:2rem;padding-top:var(--wp--preset--spacing--40);padding-bottom:var(--wp--preset--spacing--70)"><!-- wp:group {"style":{"spacing":{"blockGap":"0"}},"layout":{"type":"flex","flexWrap":"nowrap","justifyContent":"space-between","verticalAlignment":"top"}} -->
+<div class="wp-block-group"><!-- wp:group {"style":{"spacing":{"blockGap":"0"},"layout":{"selfStretch":"fill","flexSize":null}},"layout":{"type":"flex","orientation":"vertical","justifyContent":"stretch"}} -->
+<div class="wp-block-group"><!-- wp:heading {"level":1,"style":{"elements":{"link":{"color":{"text":"var:preset|color|contrast"}}},"typography":{"lineHeight":"0.8"}},"textColor":"contrast"} -->
+<h1 class="wp-block-heading has-contrast-color has-text-color has-link-color" style="line-height:0.8"><?php esc_html_e('Piccolo Teatro', 'massimo');?></h1>
+<!-- /wp:heading -->
+
+<!-- wp:site-title {"className":"no-underline"} /--></div>
```

Comment:
the site title shouldn't be an h1, since we already have h1s on the templates that are giving better information what the page is about (post title, archive title, etc) and we can't have duplicate h1s

Reply:
I was not aware I could change that in the Editor. But I can, fantastic. I just did it.